library.js

```diff
@@ -0,0 +1,68 @@
+const booksDiv = document.querySelector('.books');
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const add = document.querySelector('.add');
+/* eslint no-undef: "off" */
```

Comment:
- [ ] as the previous CR already mentioned Let's try not to disable linters as they help us write cleaner code. You can consider splitting the code in this file into different files.